style.css

```diff
@@ -0,0 +1,130 @@
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+body {
+  background-color: white;
+  margin: 0;
+  padding: 0;
+  line-height: 2rem;
+  font-family: "Poppins", sans-serif;
+  font-size: 2rem;
+  font-weight: 300;
+  color: var(--darkBlue);
+  align-items: center;
+  box-sizing: border-box;
+}
+
+h1,
+h2 {
+  margin: 4ex;
+  color-scheme: initial;
+  font-size: larger;
+}
+
+.title {
+  display: block;
+  font-family: "Poppins", sans-serif;
+  font-style: 16px;
+  font-size: large;
+  line-height: 30px;
+  text-align: center;
+}
+
+.logo_vector-btn {
+  width: 24pt;
+  height: 24pt;
+  align-items: center;
+  fill: solid rgb(red, green, rgb(7, 7, 217));
+}
+
+.header_bar {
+  display: flex;
+  justify-content: space-between;
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓